### PR TITLE
Harmonize list headers for trigger and sources

### LINKED ISSUE #567: Harmonize list headers for "trigger list", "cronjob list" and "apiserver list"
Actually the columns wrt to Sink / Subscription are different:

* CronJon and Apiserver source display the sink shortcut ('svc:myservice')
* Trigger shows the resolved URL ("SUBSCRIBER_URL")

Also, the condition and reason columns should be the same as for services. 
Should we add AGE, too ?

### PR DESCRIPTION
**Pull request by daisy-ycguo**
Fixes #567

## Proposed Changes

* Change `SUBSCRIBER_URI` to `SINK` in trigger list
* Add `CONDITIONS` to trigger list
* Add `AGE` to trigger and source ( cronjob, apiserver, binding ) list
* Move `SinkToString` to package `flags` and reuse in trigger list

### CODE REVIEW

File: pkg/kn/commands/flags/sink.go [hunk 1]
```diff
@@ -105,3 +105,18 @@ func parseSink(sink string) (string, string) {
...
 	}
 }
+
+// SinkToString prepares a sink for list output
+func SinkToString(sink duckv1.Destination) string {
+	if sink.Ref != nil {
+		if sink.Ref.Kind == "Service" {
+			return fmt.Sprintf("svc:%s", sink.Ref.Name)
+		} else {
+			return fmt.Sprintf("%s:%s", sink.Ref.Kind, sink.Ref.Name)
```
Comment (rhuss) [inline pkg/kn/commands/flags/sink.go]: can we make the kind lower case ? That's how we use it normally.

Comment (daisy-ycguo) [reply pkg/kn/commands/flags/sink.go]: Done. covert to lower case in the latest version.

**Comment by rhuss** [comment]
thanks !

/lgtm

**Pull request merged**